test/parallel/test-http2-perf_hooks.js

```diff
@@ -26,7 +26,7 @@ const obs = new PerformanceObserver(common.mustCall((items) => {
       switch (entry.type) {
         case 'server':
           assert.strictEqual(entry.streamCount, 1);
-          assert.strictEqual(entry.framesReceived, 5);
```

Comment:
This is already flaky on `master` but exacerbated by some of the changes here. I think it might be timing sensitive? Minimum seems to be 3.

Reply:
+1 for the fix, can you add a comment?